JavaScript/chapter02/2.1 - Remove Dups/solution.js

```diff
@@ -0,0 +1,71 @@
+class Node {
```

Comment:
i forgot to notice this before, but can you add your solution to a file called `aditya9061.js`?